Mobile/starter_project_matador/lib/features/example/presentation/screen/splash_screen.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:flutter/material.dart';
+import 'package:matador/features/example/presentation/widgets/background_container.dart';
+
+class SplashScreen extends StatefulWidget {
+  const SplashScreen({super.key});
+
+  @override
+  State<SplashScreen> createState() => _SplashScreenState();
+}
+
+class _SplashScreenState extends State<SplashScreen> {
+  @override
+  Widget build(BuildContext context) {
+    var size = MediaQuery.of(context).size;
+    return Container(
+      color: const Color.fromARGB(255, 233, 233, 233),
```

Comment:
if there is a way to localize color definitions and reuse them on other pages as well, I would recommend it